README.md

```diff
@@ -17,6 +17,18 @@ variables can be customised.
 -   wagtail `>=2.15` if using `WagtailBaseForm`
 -   sass `>=1.33.0` if building the sass yourself
 
+> [!NOTE] > **[govuk-frontend](https://github.com/alphagov/govuk-frontend) will
+> not, and does not need to, be installed to use this package.**
+>
+> All form-related styles from `govuk-frontend==5.4.1` have been
+> copied into this project with the prepended "govuk-" replaced with
+> "tbxforms-", e.g. `.govuk-button` to `.tbxforms-button` and
+> `@mixin govuk-clearfix` to `@mixin tbxforms-clearfix`.
+
+Installing the complete GOV.UK Frontend increases the bundle size, which is
+unnecessary for most projects. While this increases the bundle size for
+government projects, the former outweighs the latter.
```

Comment:
How about this? (former/latter doesn't quite make sense here I think)

```
Installing the original GOV.UK Frontend package increases the bundle size, which is
unnecessary for most projects, but it might be justified for some government projects.
```

Reply:
Updated in https://github.com/torchbox/tbxforms/pull/80/commits/4c775273f39a90407822090cbb9c1c737d919cf0